src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -240,68 +240,81 @@ const ContentListTable = () => {
   } = data;
 
   const rowActions = useCallback(
-    (rowData: ContentItem): IAction[] =>
-      isRedHatRepository
-        ? features?.snapshots?.accessible
-          ? [
+    (rowData: ContentItem): IAction[] => {
+      const triggerSnapshotAction = {
+        isDisabled: actionTakingPlace || rowData?.status === 'Retrying',
+        title: 'Trigger Snapshot',
+        onClick: async () => {
+          try {
+            await introspectRepository({ uuid: rowData?.uuid, reset_count: false });
```

Comment:
so snapshotting an intropsecting are two different things.  We already have an action in the kaba to introspect the repo using an existing api.  Here we want to trigger the snapshot endpoint:  POST /api/content_sources/v1.0/repositories/$UUID/snapshot/

You'll need to create a new api method similar to this: https://github.com/content-services/content-sources-frontend/blob/main/src/services/Content/ContentApi.ts#L323-L331 but for snapshotting

Reply:
Fixing now